File: 프로그래머스/0/120892. 암호 해독/암호 해독.js
function solution(cipher, code) {
    let answer = "";
    let arr_cipher = cipher.split("");
    // cipher를 arr로 변환한다
    
    for (let i = 0; i < arr_cipher.length; i++) {
        if ((i + 1) % code === 0) {
            answer += arr_cipher[i];
        } // i가 0일때, 배열의 인덱스가 0부터 시작하기 때문에 
          // i에 1을 더한 값을 code로 나누어 나머지가 0인 경우를 찾는다
          // i가 code의 배수 번째인 경우, arr_cipher[i]의 값을 answer에 추가한다
    }
    return answer;
}

File: 프로그래머스/lv0/120892. 암호 해독/암호 해독.java
class Solution {
    public String solution(String cipher, int code) {
        String answer = "";
        String[] arr_cipher = cipher.split("");
        // cipher를 arr로 변환한다
        
        for(int i = 0; i < arr_cipher.length+ 1; i++) {
        	if(i * code == 0) {
        		continue;
        	} // 0번째는 code에 해당하지 않으므로 제외한다
        	if(i % code == 0) {
                answer += arr_cipher[i - 1];
            } // code의 배수에 해당하는 원소들은 실제 code 배수번째 원소의
              // 다음 원소이므로 i - 1로 위치를 조정해준다
        }
        return answer;
    }
}